postings/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import PostingDeatilView
+
+urlpatterns=[
+    path('/detail/<int:posting_id>', PostingDeatilView.as_view()),
```

Comment:
```suggestion
    path('/<int:posting_id>', PostingDeatilView.as_view()),
```
detail 없이도 충분히 상품 정보 1개라는 의미를 가지고 있습니다.